traffic_ops/testing/api/README.md

```diff
@@ -41,7 +41,31 @@ In order to run the tests you will need the following:
     and here: `traffic-ops-test.conf` 
 
     The Traffic Ops users will be created by the tool for accessing the API once the database is accessible.
-
+    
+    Note that for the database to successfully set up the tables and run the migrations, you will need `goose`.
+    On your local machines(Macs), DO NOT install `goose` using `homebrew`. It will install another non compliant
```

Comment:
I wouldn't call it non-compliant, it's just the more popular [pressly/goose](https://github.com/pressly/goose) fork instead of the unmaintained https://bitbucket.org/liamstask/goose version that we use. Can this change to saying that Homebrew installs an unsupported goose version, or that Homebrew installs a different "goose"?

Reply:
Fixed